timeago/lib/src/messages/hi_messages.dart

```diff
@@ -0,0 +1,71 @@
+import 'package:timeago/src/messages/lookupmessages.dart';
+
+class EnMessages implements LookupMessages {
```

Comment:
`EnMessages` and `EnShortMessages` should be renamed to `HiMessages` and `HiShortMessages`